jenkins/deployment/community/community-build-cortx.groovy

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-k8-deployment-node'
+        }
+    }
+    
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
+
+    parameters {
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for CORTX Cluster scripts', trim: true)
```

Comment:
Please take `CORTX_RE_REPO` also as input

Reply:
Done. But where we will use it? @shailesh-vaidya

Reply:
I guess we will need it at `url: "${CORTX_RE_REPO}"` in https://github.com/Abhirup-A-G/cortx-re/blob/CORTX-30171-updated/jenkins/deployment/community/community-build-cortx.groovy#L34